bundle/src/main/java/com/adobe/acs/commons/wcm/properties/shared/impl/SharedComponentPropertiesBindingsValuesProvider.java

```diff
@@ -57,107 +55,81 @@
 public class SharedComponentPropertiesBindingsValuesProvider implements BindingsValuesProvider {
     private static final Logger log = LoggerFactory.getLogger(SharedComponentPropertiesBindingsValuesProvider.class);
 
-    @Reference
-    private PageRootProvider pageRootProvider;
-
-    @Reference
+    /**
+     * Bind if available, check for null when reading.
+     */
+    @Reference(policyOption = ReferencePolicyOption.GREEDY, cardinality = ReferenceCardinality.OPTIONAL_UNARY)
```

Comment:
Was this a change so that using a SCP annotation on a model w/o SCP configured wont blow things up?

Reply:
Yes, exactly. I want the annotation behavior to "failover gracefully" to the default bindings, like I'm doing with @SharedValueMapValue behaving like @ValueMapValue.